pkg/cache/cache.go

```diff
@@ -76,24 +77,30 @@ func newCohort(name string, size int) *Cohort {
 	}
 }
 
+// ResourceFlavor the internal representation of the API.
+type ResourceFlavor struct {
+	Obj *kueue.ResourceFlavor
+	// The set of QueueClusters referencing this flavor
+	QueueClusters sets.String
```

Comment:
ClusterQueues?

But since this struct doesn't seem to be used in this PR, I would leave it for the next.

Reply:
yeah, removed, I was starting to sketch an implementation but forgot to remove this. We will likely not need this anyways.